src/SqlBindingUtilities.cs

```diff
@@ -60,7 +60,14 @@ public static string GetWebSiteName(IConfiguration configuration)
             {
                 throw new ArgumentNullException(nameof(configuration));
             }
-            return configuration.GetConnectionStringOrSetting(SqlBindingConstants.WEBSITENAME);
+            string websitename = configuration.GetConnectionStringOrSetting(SqlBindingConstants.WEBSITENAME);
+            // We require a WEBSITE_SITE_NAME for avoiding duplicates if users use the same function name accross apps.
+            if (string.IsNullOrEmpty(websitename))
+            {
+                throw new ArgumentException(websitename == null ? $"WEBSITE_SITE_NAME setting is missing in your function app settings, please add the setting with a string value. Please refer to https://github.com/Azure/azure-functions-sql-extension/blob/main/docs/BindingsOverview.md#website_site_name for more information." :
```

Comment:
Would suggest putting the link in both. And in fact, I'd just simplify this into one and say something like "WEBSITE_SITE_NAME can't be null or empty in your function app settings, please add..."